### PR TITLE
provide detailed alt text to cond image 01.

### PR DESCRIPTION
**Pull request by blankdots**
Flowchart conditional image now has a detailed description: ../fig/python-flowchart-conditional.png

Related to https://github.com/swcarpentry/python-novice-inflammation/issues/781

Note: Followed the recommendations at: https://webaim.org/techniques/alttext/#basics trying to be as specific as possible for the content and the function.

- [x] `07-cond.md > Executing a Conditional`

### CODE REVIEW

File: _episodes/07-cond.md [hunk 1]
```diff
@@ -53,7 +53,8 @@ If the test is false,
 the body of the `else` is executed instead.
 Only one or the other is ever executed:
 
-![Executing a Conditional](../fig/python-flowchart-conditional.png)
+![If-Else flow diagram for num variable testin if it is greather than 100.
+If true it prints 'greater' and 'done', if false it prints 'not greather' and 'done'.](../fig/python-flowchart-conditional.png)
```
Comment (ldko) [inline _episodes/07-cond.md]: Another possibility to consider is modifying the text before the figure slightly in order to reduce the `alt` attribute text. For example:

```
The second line of this code uses the keyword `if` to tell Python that we want to make a choice.
If the test that follows the `if` statement is true,
the body of the `if`
(i.e., the set of lines indented underneath it) is executed, and "greater" is printed.
If the test is false,
the body of the `else` is executed instead, and "not greater" is printed.
Only one or the other is ever executed before continuing on with program execution to print "done":

![If-Else flow diagram testing if variable num is greater than 100]
```

Comment (blankdots) [reply _episodes/07-cond.md]: good point, changed to that.

File: _episodes/07-cond.md [hunk 2]
```diff
@@ -48,12 +48,12 @@ done
...
-(i.e., the set of lines indented underneath it) is executed.
+(i.e., the set of lines indented underneath it) is executed, and "greater" is printed.
 If the test is false,
-the body of the `else` is executed instead.
-Only one or the other is ever executed:
+the body of the `else` is executed instead, and "not greater" is printed.
+Only one or the other is ever executed before continuing on with program execution to print "done":
 
-![Executing a Conditional](../fig/python-flowchart-conditional.png)
+![If-Else flow diagram testing if variable num is greater than 100](../fig/python-flowchart-conditional.png)
```
Comment (maxim-belkin) [inline _episodes/07-cond.md]: What do you guys think about this alt. text ?
> A flowchart diagram of the if-else construct that tests if variable num is greater than 100

Comment (ldko) [reply _episodes/07-cond.md]: That sounds fine to me. In your version it is more clear that it is the if-else construct testing num > 100 vs. the diagram doing any actual testing.

Comment (blankdots) [reply _episodes/07-cond.md]: @maxim-belkin changed to that, thx!

**Comment by ldko** [comment]
Thanks for doing all the follow up on this one, @blankdots

**Pull request merged**